sml_builder/page.py

```diff
@@ -27,6 +27,20 @@ def cookies_page():
     return render_template("cookies.html")
 
 
+@app.route("/accessibility-statement")
+def accessibility_page():
+    try:
+        with open(
+            "./content/accessibility/accessibility-statement.md", "r", encoding="utf-8"
+        ) as input_file:
+            text = input_file.read()
+            escaped_text = escape(text)
+            body = Markup(markdown.markdown(escaped_text))
+    except OSError as e:
+        _page_not_found(e)
```

Comment:
quick question is this a specific function?

Reply:
Yes, it's a function that we have. We call it to display a 404 page not found.

Reply:
Nice one thanks